### ISSUE TITLE
Profile Operation Dressups

### ISSUE THREAD

**Issue by giannissc** [voneiden/cq-cam#58]
- [ ] Overcut
    - [ ] Dogbone
    - [ ] T-bone Long
    - [ ] T-bone Short
    - [ ] T-bone Vertical
    - [ ] T-bone Horizontal 
- [ ] Dragknife

**Comment by voneiden** [comment]
Dogbones are definitely on my wishlist :+1: 

But the time is not right to start working on these anytime soon. Plenty of work to do in getting the basics right with cq-cam.

**Comment by giannissc** [comment]
Yeah I know it just so that I don't forget and we can create milestones to prioritize them!
